lib/Command/Signaling/ListCommand.php

```diff
@@ -0,0 +1,67 @@
+<?php
+declare(strict_types=1);
+/**
+ * @copyright Copyright (c) 2018 Denis Mosolov <denismosolov@gmail.com>
+ *
+ * @author Denis Mosolov <denismosolov@gmail.com>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace OCA\Spreed\Command\Signaling;
+
+use OCP\IConfig;
+use OC\Core\Command\Base;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class ListCommand extends Base {
+
+	/** @var IConfig */
+	private $config;
+
+	/**
+	 */
+	public function __construct(IConfig $config) {
+		parent::__construct();
+		$this->config = $config;
+	}
+
+	protected function configure() {
+		$this
+			->setName('talk:signaling:list')
+			->setDescription('List external signaling servers.')
+			->addOption(
+				'output',
```

Comment:
this should not be necessary, just call `parent::configure()` instead, see e.g. https://github.com/nextcloud/server/blob/0eebff152a177dd59ed8773df26f1679f8a88e90/core/Command/App/GetPath.php#L34